jaeger-core/src/test/java/com/uber/jaeger/propagation/FaultyTextMap.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016, Uber Technologies, Inc
```

Comment:
For new files, the header should be `The Jaeger Authors` (the year is wrong as well, it seems).